odoo/addons/base/tests/test_module.py

```diff
@@ -0,0 +1,88 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import os.path
+import tempfile
+from os.path import join as opj
+from unittest.mock import patch
+
+import odoo.addons
+from odoo.modules.module import _load_manifest
+from odoo.release import major_version
+from odoo.tests.common import BaseCase
+
+
+class TestModuleManifest(BaseCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._tmp_dir = tempfile.TemporaryDirectory(prefix='odoo-test-addons-')
+        cls.addClassCleanup(cls._tmp_dir.cleanup)
+        cls.addons_path = cls._tmp_dir.name
+
+        patcher = patch.object(odoo.addons, '__path__', [cls.addons_path])
+        patcher.start()
+        cls.addClassCleanup(patcher.stop)
+
+    def setUp(self):
+        self.module_root = tempfile.mkdtemp(prefix='odoo-test-module-', dir=self.addons_path)
+        self.module_name = os.path.basename(self.module_root)
+
+    def write_manifest(self, **overrides):
+        mandatories = {'name': f'Temp {self.module_name}', 'license': 'MIT'}
+        with open(opj(self.module_root, '__manifest__.py'), 'w') as manifest_fd:
+            manifest_fd.write(str(dict(mandatories, **overrides)))
+
+    def test_default_manifest(self):
+        self.write_manifest()
+        with self.assertNoLogs('odoo.modules.module', 'WARNING'):
+            manifest = _load_manifest(self.module_name)
+        self.maxDiff = None
+
+        self.assertDictEqual(manifest, {
+            'addons_path': self.addons_path,
+            'application': False,
+            'assets': {},
+            'author': 'Odoo S.A.',
+            'auto_install': False,
+            'bootstrap': False,
+            'category': 'Uncategorized',
+            'data': [],
+            'demo': [],
+            'demo_xml': [],
+            'depends': [],
+            'description': '',
+            'external_dependencies': [],
+            'icon': '/base/static/description/icon.png',
+            'init_xml': [],
+            'installable': True,
+            'images': [],
+            'images_preview_theme': {},
+            'license': 'MIT',
```

Comment:
Looks wrong. A warning was added when loading manifest so that devs have to define the manifest manually, especially for enterprise. 

This also makes the check on existing license almost useless when loading manifest (defaulting to LGPL-3 if falsy)